src/languageserverinstance.jl

```diff
@@ -117,7 +117,17 @@ function setdocument!(server::LanguageServerInstance, uri::URI2, doc::Document)
 end
 
 function deletedocument!(server::LanguageServerInstance, uri::URI2)
+    doc = getdocument(uri)
+
     delete!(server._documents, uri)
+
+    for d in getdocuments_value(server)
+        if d.root===doc
+            d.root = nothing
+            # TODO Check whether we need to do something else here as well,
+            # maybe rerun the linter or something like that?
```

Comment:
@ZacLN are we sure we don't need to do antyhing like that here?

Reply:
Sorry, I missed the comment. Yes we do, we want to run a scope pass on these files. `scopepass(d, d)` No other special handling should be required